config/locales/en.yml

```diff
@@ -661,12 +661,12 @@ en:
           driving_school: "Driving School"
           embassy: "Embassy"
           events_venue: "Events Venue"
-          fast_food: "Fast Food"
+          fast_food: "Fast Food Restaurant"
```

Comment:
This should be removed, since amenity=fast_food also covers food trucks and takeaway only outlets.